RecoTracker/TkDetLayers/src/Phase2OTtiltedBarrelLayer.h

```diff
@@ -0,0 +1,49 @@
+#ifndef TkDetLayers_Phase2OTtiltedBarrelLayer_h
+#define TkDetLayers_Phase2OTtiltedBarrelLayer_h
+
+
+#include "TrackingTools/DetLayers/interface/RodBarrelLayer.h"
+#include "Phase2OTBarrelLayer.h"
+#include "Phase2OTBarrelRod.h"
+#include "Phase2OTEndcapRing.h"
+#include "SubLayerCrossings.h"
+
+
+/** A concrete implementation for Phase2OTtiltedBarrel layer 
+ *  built out of BarrelPhase2OTBarrelRod
+ */
+
+#pragma GCC visibility push(hidden)
```

Comment:
"namespace {"  defines it in the context of the file (equivalent to inline)
hidden visibility in the context of shared library